articles/concepts/pauli-measurements.md

```diff
@@ -1,120 +1,215 @@
 
 ## The No-Cloning Theorem
-Quantum information is powerful.  It enables us to do amazing things such as factor numbers exponentially faster than the best known classical algorithms, or efficiently simulate correlated electron systems that classically require exponential cost to simulate accurately.  However, there are limitations to the power of quantum computing.  One such limitation is given by the *No-Cloning Theorem*.
+
+Quantum information is powerful.
+It enables us to do amazing things such as factor numbers exponentially faster than the best known classical algorithms, or efficiently simulate correlated electron systems that classically require exponential cost to simulate accurately.
+However, there are limitations to the power of quantum computing.
+One such limitation is given by the *No-Cloning Theorem*.
 
 The No-Cloning Theorem is aptly named.
 It disallows cloning of generic quantum states by a quantum computer.
 The proof of the theorem is remarkably straightforward.
-While a full proof of the no-cloning theorem is a little too technical for our discussion here, the proof of the theorem in the case where the quantum computer in question has no additional ancilla qubits is within our scope (ancilla qubits are qubits used for scratch space during a computation and are easily used and managed in Q#, see <xref:microsoft.quantum.techniques.qubits>).
-For such a quantum computer, the cloning operation must be a unitary matrix. We disallow measurement, since it would corrupt the quantum state we are trying to clone. The unitary matrix we want must have the property that
+While a full proof of the no-cloning theorem is a little too technical for our discussion here, the proof of the theorem in the case where the quantum computer in question has no additional auxiliary qubits is within our scope (auxiliary qubits are qubits used for scratch space during a computation and are easily used and managed in Q#, see <xref:microsoft.quantum.techniques.qubits>).
+For such a quantum computer, the cloning operation must be described by a unitary matrix.
+We disallow measurement, since it would corrupt the quantum state we are trying to clone.
+We want the unitary matrix used to simulate our cloning operation to have the property that
 $$
-  U \ket{\psi} \ket{0} = \ket{\psi} \ket{\psi},
+  U \ket{\psi} \ket{0} = \ket{\psi} \ket{\psi}
 $$
 for any state $\ket{\psi}$.
 The linearity property of matrix multiplication then implies that for any second quantum state $\ket{\phi}$,
 
+$$
 \begin{align}
 &U\left[\frac{1}{\sqrt{2}}\left(\ket{\phi}+\ket{\psi} \right)\right]\ket{0}=\frac{1}{\sqrt{2}}U\ket{\phi}\ket{0}+\frac{1}{\sqrt{2}}U\ket{\psi}\ket{0}\\\\
 &\qquad\qquad=\frac{1}{\sqrt{2}}\left(\ket{\phi}\ket{\phi}+\ket{\psi}\ket{\psi}\right)\\\\
 &\qquad\qquad\ne \left(\frac{1}{\sqrt{2}}\left(\ket{\phi}+\ket{\psi} \right)\right)\otimes\left(\frac{1}{\sqrt{2}}\left(\ket{\phi}+\ket{\psi} \right)\right).
 \end{align}
+$$
```

Comment:
Switch the align "&"s to be at the equal signs?

Reply:
Yep, good catch. Done.